packages/md-enhance/src/client/components/playground/PlaygroundInternal.ts

```diff
@@ -0,0 +1,132 @@
+import { defineComponent, h, onMounted, ref, reactive } from "vue";
+import type { VNode } from "vue";
+
+import type { ReplProps } from "@vue/repl";
+import { ReplStore, Repl } from "@vue/repl";
```

Comment:
I think this component may might be large, so you may dynamic import this or the whole component. Vue3 provides a defineAsyncComponent